blockstore/noop.go

```diff
@@ -0,0 +1,66 @@
+package blockstore
+
+import (
+	"context"
+	"io"
+
+	blocks "github.com/ipfs/go-block-format"
+	cid "github.com/ipfs/go-cid"
+)
+
+var _ Blockstore = (*noopstore)(nil)
+
+type noopstore struct {
```

Comment:
nit: discardstore? It's not exactly a no-op.

Reply:
Dunno, I am not married to the name. If you think it's better, then we can rename :)

Reply:
I don't _really_ care. I'd just expect a noop store to always do nothing (i.e., never return anything)

Reply:
Yeah, agreed. I'll call it `discardstore` and change the config option name to "discard".

Reply:
Done in https://github.com/filecoin-project/lotus/pull/6474/commits/c0a1cfffa1f1fbea767af4f214fbefde14187622